pyseir/deployment/model_conversion_factors.py

```diff
@@ -0,0 +1,102 @@
+import us
+import structlog
+from datetime import timedelta
+
+from pyseir import load_data
+
+log = structlog.get_logger()
+
+UNITY_SCALING_FACTOR = 1
+
+
+def _get_model_to_dataset_conversion_factors_for_state(state, t0_simulation, fips):
```

Comment:
add some type annotations, state needs to be a string that's either a full name or a valid abbreviation.